delete.php

```diff
@@ -60,4 +60,16 @@
   die('?view&s='.$server['id'].'&d='.$server['db']);
 }
 
+if (isset($_GET['batch_del'])){
+  $keys = $_POST['checked_keys'];
+  $keys = trim($keys,',');
+  $keys = explode(',', $keys);
+  if ($keys){
+    foreach ($keys as $key) {
+      $redis->del($key);
+    }
+  }
+  die('?view&s='.$server['id'].'&d='.$server['db']);
```

Comment:
Please add `.'&key='.urlencode($keys[0])` at the end here.
And while you're doing that please do the same with the `die` statement above. They currently both redirect to a page that just says `Invalid key` which shouldn't happen.

Reply:
Also make sure `count($keys) > 0` otherwise you can do `die('No keys to delete`);`.

Reply:
> Also make sure `count($keys) > 0` otherwise you can do `die('No keys to delete`);`.

i use `if (empty($keys)) die('No keys to delete');` ,beacause `explode(',', $keys);` always return a not empty array